momentum/3/3-api/arch-primary-apis.md

```diff
@@ -297,6 +297,8 @@ A hook is made up of a number of elements:
 
     *   Third – The invocation of the hook's providers. This always includes a special first argument called `EC_HOOK_CLOSURE` and then the variable names (in order) defined in the caller function shown in the [log_permanent_failure_v1 example](/momentum/3/3-api/arch-primary-apis#arch.using.hooks.example). `EC_HOOK_CLOSURE` just refers to the closure argument that was passed when the provider was registered. It's a symbolic name used by the programmer within the macro in order to obscure the 'real' name.
 
+    When writing hook implementations, the first argument in many hooks is `void *closure`. The closure is passed into hooks written in C, but for hooks written in lua, the closure argument is not passed, and instead arguments start with the second argument.
```

Comment:
Super minor/nit picky but "lua" should be capitalized like it is down in hooks-core-generate-rfc-3464-message.md and missing a comma after "instead"